doc/classes/CanvasItem.xml

```diff
@@ -648,7 +648,7 @@
 		</signal>
 		<signal name="hidden">
 			<description>
-				Emitted when the [CanvasItem] becomes hidden, either because its own [member visible] property was set to [code]false[/code] or the [member visible] property of one of its [CanvasItem] ancestors was set to [code]false[/code].
+				Emitted when the [CanvasItem] is no longer visible in the tree (see [method is_visible_in_tree]).
```

Comment:
A bit iffy on no longer mentioning "hidden", and letting the signal's name imply it. It makes it look complicated, but it's also not as good to localize.
```suggestion
				Emitted when the [CanvasItem] is hidden, that is, it's no longer visible in the tree (see [method is_visible_in_tree]).
```
Or something like this.

Reply:
Right! Agreed. I personally find `, that is, ` a bit awkward, so I opted for: 

`Emitted when the [CanvasItem] is hidden, or more specifically, no longer visible in the tree (see [method is_visible_in_tree]).`

Reply:
@Mickeon Do we dislike/avoid abbreviations like `i.e.` in the docs?

I think `Emitted when the [CanvasItem] is hidden, i.e. no longer visible in the tree (see [method is_visible_in_tree]).` reads nice, no? :thinking:

Reply:
> Do we dislike/avoid abbreviations like i.e. in the docs?

No, not at all, actually. KoBeWi uses it all the time. I do dislike it personally for a few reasons, though, but not to the point where I'd outright ban it.

Reply:
@AdriaandeJongh With that said I think you should go with @kleonc 's suggestion.

Thank you so much for keeping up with our annoying antics.